cachito/web/config.py

```diff
@@ -17,6 +17,8 @@ class Config(object):
     CACHITO_LOG_LEVEL = "INFO"
     CACHITO_LOG_FORMAT = "[%(asctime)s %(name)s %(levelname)s %(module)s.%(funcName)s] %(message)s"
     CACHITO_MAX_PER_PAGE = 100
+    # Pairs of mutually exclusive package managers (cannot process the same package)
+    CACHITO_MUTUALLY_EXCLUSIVE_PACKAGE_MANAGERS = [("npm", "yarn")]
```

Comment:
Should we add some validation for this configuration to ensure the inner tuple/lists always have 2 elements?

Reply:
Good point, added

Reply:
Do you want `("gomod", "git-submodule")` here? It seems like it would be more important to have the mutually exclusive pair on the default class than it would on the `DevelopmentConfig`.

Reply:
@arewm this is intentional, to differentiate between `("npm", "yarn")`, which will be exclusive by design and `("gomod", "git-submodule")`, which are exclusive until we implement support. In the deployment playbooks, this will be configured properly